patches/minecraft/net/minecraft/client/renderer/culling/Frustum.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/client/renderer/culling/Frustum.java
++++ b/net/minecraft/client/renderer/culling/Frustum.java
+@@ -59,6 +_,8 @@
+    }
+ 
+    public boolean m_113029_(AABB p_113030_) {
++      //FORGE: exit early for infinite bounds, these would otherwise fail in the intersection test at certain camera angles
```

Comment:
Space between the comment marker and `FORGE`. Also, I'm thinking whether to include the issue number, for future reference.

```suggestion
+      // FORGE: exit early for infinite bounds, these would otherwise fail in the intersection test at certain camera angles (GH-9321)
```